testproject/testapp/tests.py

```diff
@@ -616,3 +625,73 @@ def test_put_should_update_user(self):
         self.assert_status_equal(response, status.HTTP_200_OK)
         user = utils.refresh(user)
         self.assertEqual(data['email'], user.email)
+
+
+class SerializersManagerTest(SimpleTestCase):
+
+    def test_serializer_manager_init(self):
+        serializers_manager = djoser.serializers.SerializersManager({})
+        self.assertFalse(serializers_manager.serializers)
+
+    def test_get_serializer(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer})
+        serializer_class = serializers_manager.get('user')
+        self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
+
+    def test_get_serializer_non_proper_name(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer
+        })
+        self.assertRaises(Exception, serializers_manager.get, 'bad_name')
+
+    def test_get(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer})
+        serializer_class = serializers_manager.get('user')
+        self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
```

Comment:
Good point. Is exactly the same, because of refactoring SerializeManager. Now it is redundant. I'll remove one of this tests.